Immediate serveress iterations (#10659)
DataDog/documentation: content/en/serverless/installation/nodejs.md

{{< img src="serverless/serverless_monitoring_installation_instructions.png" alt="Instrument AWS Serverless Applications"  style="width:100%;">}}

> If you previously set up Datadog Serverless using the Datadog Forwarder, see the [installation instructions here][2]. Currently, Lambda only supports sending data to Datadog in the `us-east-1`, `eu-west-1`, and `eu-south-1` regions.

Reviewer: ```suggestion
If you previously set up Datadog Serverless using the Datadog Forwarder or your Lambda functions are not deployed to the `us-east-1`, `eu-west-1` or `eu-south-1` regions, see the [installation instructions here][2].
```
